build.rs

```diff
@@ -26,6 +26,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let bindings = bindgen::Builder::default()
         .header("littlefs/lfs.h")
         .clang_arg(format!("--target={}", target))
+        .clang_arg("-I/usr/arm-none-eabi/include")
```

Comment:
I believe this should be documented. I had a compiler just not at this location so I had to make a symlink `ln -s` for this to work.

Reply:
Ideally I would prefer something that does not need that kind of hack and works even on weird distros.

Reply:
I'm not sure what are the options honestly as I'm not experienced with C/C++ but I'm happy to say that so far this version is working with my initial testing.